### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by romanstrobl**
Merge of all changes for release `1.0.0` from `develop` into `master`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,28 +8,28 @@
 
     <groupId>io.getlime.security</groupId>
     <artifactId>powerauth-admin-parent</artifactId>
-    <version>0.24.0</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (petrdvorak) [inline pom.xml:11]: This should probably be already changed to `1.0.0` in this PR?

**Review by petrdvorak — APPROVED** [review]
Looks OK but we should probably already change the version number in this "prepare 1.0.0" PR...

File: pom.xml (hunk 1 above)
Comment (romanstrobl) [reply pom.xml]: This change was included in another pull request and was fixed by merging it.

**Comment by romanstrobl** [comment]
I'll wait with merging the pull request for release of PowerAuth server. The build is failing because of missing dependency on this release.

**Review by hvge — APPROVED** [review]
Looks OK

**Pull request merged**